app/workers/check_all_editions_links_worker.rb

```diff
@@ -0,0 +1,29 @@
+# Calls the Link Checker API to verify all links in public editions, either per organisation or not
+class CheckAllEditionsLinksWorker
```

Comment:
I would make this class `CheckOrganisationLinksWorker` and make it a requirement to pass in an organisation ID, since users can use the `CheckOrganisationLinksScheduledWorker` worker if they want to check all the links.